src/nvim/testdir/test_options.vim

```diff
@@ -621,6 +621,26 @@ func Test_local_scrolloff()
   set siso&
 endfunc
 
+func Test_writedelay()
+  if !has('reltime')
+    return
+  endif
+  new
+  call setline(1, 'empty')
+  redraw
+  set writedelay=10
+  let start = reltime()
+  call setline(1, repeat('x', 70))
+  redraw
+  let elapsed = reltimefloat(reltime(start))
+  set writedelay=0
+  " With 'writedelay' set should take at least 30 * 10 msec
+  " call assert_inrange(30 * 0.01, 999.0, elapsed)
+  call assert_inrange(0.01, 999.0, elapsed)
```

Comment:
@bfredl Is this suppose to happen?

Reply:
'writedelay' works quite differently in neovim so not necessarily.